index.html

```diff
@@ -13,53 +13,162 @@
+                </ul>
+            <a class="logo" href="index.html"> <img src="images/MyLogo.jpg" alt=""></a>
+            </nav>   
         </header>
-    </div>
-    <br>
-
-    <section class="main">
-        <p>
-            Im a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        <div class="contenedor">
+        <header>
+            <h1>Im Lucca
+                 Glad to see
+                  you!</h1>
+        </header>
+        </div>
+        <div class="main"><p>
+        Im a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
         </p>
+        </div>
+        <p class="let"><img src="images/LETSCONNECT.jpg" alt=""></p>
+        <div>
+            <ul>
+                <li> <a href="https://twitter.com/LuccaMicale_"><img class="twitter" src="images/Vectortw.jpg" alt=""></a></li>
+                <li> <a href="#"><img class="letteri" src="images/VectorI.png" alt=""></a></li>
+                <li> <a href="#"><img class="lettern" src="images/Vectorn.jpg" alt=""></a></li>
+                <li> <a href="#"><img class="letterm" src="images/Vectorm.jpg" alt=""></a></li>
+                <li> <a href="https://github.com/luccamicale"><img class="git" src="images/Vectorgit.jpg" alt=""></a></li>
+                <li> <a href=""><img class="hand" src="images/Vectorhnad.jpg" alt=""></a></li>
+            </ul>
+        </div>
+    </section>
+
+    <section class="all1">
+        <img class="one" src="images/SnapshootPortfolio(2).jpg" alt="">
+        <h2 class="tonic">Tonic</h2>
+        <div class="list">
+            <ul class="inline">
+                <li><img class="words" src="images/Frame2.jpg" alt=""></li>
+            </ul>
+        </div>
+        <h3 class="subtitle">A daily selection is privaly 
+            personalized reads; 
+            no accounts or sign-ups required
+        </h3>
+        <div class="leng">
+            <ul>
+                <li><img class="lenguage" src="images/Tags.jpg" alt=""></li>
+                <li><a href="https://github.com/luccamicale"><img class="see" src="images/Enabled.jpg" alt=""></a></li>
+            </ul>
+        </div>
+    </section>
+
+    <section class="all2">
+        <img class="two" src="images/SnapshootPortfolio)1.jpg" alt="">
+        <h2 class="multipost">Multi - Post 
+            Stories
+        </h2>
+        <div class="list2">
+            <ul class="inline2">
+                <li><img class="words2" src="images/Frame2.jpg" alt=""></li>
+            </ul>
+        </div>
+        <h3 class="subtitle1">A daily selection is privaly 
+            personalized reads; 
+            no accounts or sign-ups required
+        </h3>
```

Comment:
Great job @luccamicale 🤝

- [ ] Kindly use a `p` tag for all these `h3`s. for all individual works sections.